spec/features/admin/dashboard_spec.rb

```diff
@@ -13,5 +13,39 @@
       expect(page).to have_content("Admin Dashboard")
       expect(page).to have_button("Admin")
     end
+
+    it 'displays innings section if innings are present' do
+      inning1 = create(:inning, :is_current)
+      visit admin_path
+
+      expect(page).to have_content("Innings")
+    end
+
+    it 'does not display innings section if there are no innings present' do
+      visit admin_path
+
+      expect(page).to_not have_content('Innings')
+    end
```

Comment:
We won't need this test per my above comments

Reply:
Removed